website/docs/concepts/agents.mdx

```diff
@@ -3,6 +3,9 @@ title: "Agents"
 description: "Yew's Actor System"
 ---
 
+import Image from '@theme/IdealImage';
+import agentLifeCycle from '/img/agent-life-cycle.png'
```

Comment:
This is the way to use the newly added `ideal-image` plugin and images can be added by doing
`<Image img={agentLifeCycle}/>`